src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/LetterAddressHelper.java

```diff
@@ -1,18 +1,18 @@
 package uk.gov.hmcts.reform.finrem.caseorchestration.helper;
 
-import org.springframework.stereotype.Component;
-
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 
-@Component
 public class LetterAddressHelper {
 
-    private List<String> addressLines = new ArrayList<>();
+    private LetterAddressHelper() {
```

Comment:
I think we agreed in a previous PR to use annotations to avoid having to do this

Reply:
yes, as far as I remember you and John are working on it now. Please update :)